src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days < 3) {
+    return totalCost;
+  } else if (days >= 3 && days <= 6) {
+    return (totalCost = totalCost - 20);
+  } else if (days >= 7) {
+    return (totalCost = totalCost - 50);
```

Comment:
Avoid using literals in expressions. Instead, define constants at the beginning of your function to hold these values. This will make your code more readable and maintainable. For instance, create constants for the minimum days required for a discount, and for the discount amounts themselves.

Reply:
agree